modular_skyrat/modules/cortical_borer/code/cortical_borer.dm

```diff
@@ -105,11 +105,12 @@ GLOBAL_LIST_EMPTY(cortical_borers)
 										/datum/reagent/medicine/morphine,
 										/datum/reagent/medicine/inacusiate,
 										/datum/reagent/medicine/oculine,
+										/datum/reagent/medicine/mannitol,
```

Comment:
why not neurine

Reply:
I feel like the original balance of having mannitol but needing to learn neurine was fine